wagtail/admin/templates/wagtailadmin/account/change_avatar.html

```diff
@@ -0,0 +1,27 @@
+{% extends "wagtailadmin/base.html" %}
+{% load avatar i18n %}
+
+{% block titletag %}{% trans "Change profile profile" %}{% endblock %}
+{% block content %}
+    {% trans "Change user profile" as change_str %}
+    {% include "wagtailadmin/shared/header.html" with title=change_str %}
+
+    <div class="nice-padding">
+        <form action="{% url 'wagtailadmin_account_change_avatar' %}" method="POST" enctype="multipart/form-data">
+            {% csrf_token %}
+
+                <ul class="fields">
+                    {% for field in form %}
+                        {% include "wagtailadmin/shared/field_as_li.html" with field=field %}
+                    {% endfor %}
+                </ul>
+
+            <li class="submit"><input type="submit" value="{% trans 'Update' %}" class="button" /></li>
```

Comment:
Few minor things:

* `<ul>` block should on the same level with `{% csrf_token %}` (remove 4 spaces from left)
* `<li>` without `<ul>` or `<ol>` doesn't look like a valid HTML